AGW-3860: Mule 3 proxy elements. (#189)
mulesoft/docs-api-manager: modules/ROOT/pages/proxy-mule3-elements.adoc

= Mule 3.x API Proxy elements
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

== Automatically Filtering Internal Mule HTTP Headers

> To avoid internal Mule HTTP headers from being leaked to the backend implementation or from being inlcuded in the response, the following two filters are defined:

Reviewer: typo and passive voice
```suggestion
To avoid internal Mule HTTP headers from being leaked to the backend implementation or from being included in the response, use the appropriate one of the following two filters:
```
OR
```suggestion
To avoid internal Mule HTTP headers from being leaked to the backend implementation or from being included in the response, use one of the following two filters, respectively:
```
...because there are two situations to avoid and two solutions, and order is important.
Consider separating the situations into separate texts:

To avoid...backend implementation, use the following filter:
     <filter syntax>
To avoid...from being included in the response, use the following filter:
     <filter syntax>

Reviewer 2: Done! Thanks!
